src/OutboundServer/api.yaml

```diff
@@ -165,33 +165,37 @@ paths:
               $ref: '#/components/schemas/bulkTransferRequest'
         required: true
       responses:
-        '200':
-          $ref: '#/components/responses/bulkTransferSuccess'
+        '202':
+          $ref: '#/components/responses/bulkTransferAccepted'
         '400':
           $ref: '#/components/responses/bulkTransferBadRequest'
         '500':
-          $ref: '#/components/responses/bulkTransferServerError'
-        '504':
-          $ref: '#/components/responses/bulkTransferTimeout'
+          $ref: '#/components/schemas/errorResponse'
   '/bulkTransfers/{bulkTransferId}':
-    get:
-      summary: Retrieves information for a specific bulk transfer
+    patch:
```

Comment:
The PATCH needs to change to a PUT.

Sam raised the point that only Mojaloop should raise patch notification, and a Patch should only be a notification. I.e. it should not affect a change at the hub.